indices_clear_cache.go

```diff
@@ -0,0 +1,227 @@
+package elastic
```

Comment:
The copyright notice is missing here.

Reply:
Sorry, I am not much familiar with adding copyrights. Should I add my name ?

Reply:
No. Simply copy the header from any of the other files.

Reply:
Sure ok.